modules/check_in/app/services/travel_claim/service.rb

```diff
@@ -42,17 +42,7 @@ def initialize(opts = {})
     #
     # @return [String] token
     def token
-      @token ||= begin
-        token = redis_client.token
-
-        return token if token.present?
-
-        resp = client.token
-
-        Oj.safe_load(resp.body)&.fetch('access_token').tap do |access_token|
-          redis_client.save_token(token: access_token)
-        end
-      end
+      @token ||= redis_client.token || access_token_from_veis
```

Comment:
```suggestion
      @token ||= redis_client.token.presence || access_token_from_veis
```

You probably need this to make the test pass?